DSE_5.1_Release_Notes.md

```diff
@@ -1,13 +1,38 @@
 # Release notes for DataStax Enterprise 5.1
 DSE 5.1.x is compatible with Apache Cassandra&trade; 3.11 and adds additional production-certified changes, if any. Components that are indicated with an asterisk (&ast;) (if any) are known to be updated since the prior patch version.
 
+# DataStax Enterprise 5.1.25
+21 September 2021
+
+## 5.1.26 DSE core
+* Added check against the negative value in local stream throughput (stream_throughput_outbound_megabits_per_sec) and inter dc stream throughput (inter_dc_stream_throughput_outbound_megabits_per_sec) (DB-5010)
+
+## 5.1.26 DSE SSTables
+* Backported fixes from CASSANDRA-15595 that prevent IllegalBound exceptions in tables with huge data and CASSANDRA-15869 that fixes MemoryOutputStream overflow on large bloom filters. (DB-5035)
+
+## 5.1.26 DSE Core
```

Comment:
remove here, only security - core is already above

Reply:
Fixed. Moved all CVE issues to the same sub-section and remove duplicated entries.